yarn-error.log

```diff
@@ -0,0 +1,91 @@
+Arguments: 
+  C:\Program Files\nodejs\node.exe C:\Program Files (x86)\Yarn\bin\yarn.js run coverage
+
+PATH: 
+  C:\Program Files (x86)\Intel\Intel(R) Management Engine Components\iCLS\;C:\Program Files\Intel\Intel(R) Management Engine Components\iCLS\;C:\WINDOWS\system32;C:\WINDOWS;C:\WINDOWS\System32\Wbem;C:\WINDOWS\System32\WindowsPowerShell\v1.0\;C:\Program Files (x86)\Intel\Intel(R) Management Engine Components\DAL;C:\Program Files\Intel\Intel(R) Management Engine Components\DAL;C:\Program Files (x86)\Intel\Intel(R) Management Engine Components\IPT;C:\Program Files\Intel\Intel(R) Management Engine Components\IPT;C:\WINDOWS\System32\OpenSSH\;C:\Program Files\Git\cmd;C:\Program Files\nodejs\;C:\Program Files (x86)\Yarn\bin\;C:\Users\christian\AppData\Local\Microsoft\WindowsApps;;C:\Users\christian\AppData\Local\Programs\Microsoft VS Code\bin;C:\Users\christian\AppData\Roaming\npm;C:\Users\christian\AppData\Local\Yarn\bin;C:\Program Files\heroku\bin
+
+Yarn version: 
+  1.17.3
+
+Node version: 
+  10.16.0
+
+Platform: 
+  win32 x64
+
+Trace: 
+  SyntaxError: C:\Users\christian\booked-up-fe\package.json: Unexpected token < in JSON at position 1280
```

Comment:
What is this error for?  Is that what you fixed in later commits?

Reply:
"jest": {
    "collectCoverageFrom": [
      "**/*.{js,jsx}", _<--- If you don't have .jsx files in your project this line should look like this._ "**/*.js",
      "!**/node_modules/**",
      "!**/vendor/**",
      "!**/__tests__/**"
    ]
  }

I forgot to remove the text with the arrow from above code

Reply:
Ok sounds good, resolving